src/scripts/_scoreboard.ts

```diff
@@ -0,0 +1,17 @@
+const scoreboard = document.querySelector('.js-scoreboard') as HTMLDivElement;
+const playerResult = document.querySelector(
+	'.js-player-score'
+) as HTMLParagraphElement;
+const computerResult = document.querySelector(
+	'.js-computer-score'
+) as HTMLParagraphElement;
+
+export const showScoreboard = (computerScore: number, playerScore: number) => {
+	playerResult.textContent = `${playerScore}`;
+	computerResult.textContent = `${computerScore}`;
```

Comment:
Jeżeli nie dodajesz żadnego tekstu statycznego do tej zmiennej i tej powyżej, to nie ma potrzeby opakowywania tego w `${ }`